composer.json

```diff
@@ -41,10 +41,11 @@
         "fakerphp/faker": "~1.10",
         "google/cloud-translate": "^1.6",
         "laravel/pint": "1.16.0",
-        "mockery/mockery": "^1.3.3",
+        "mockery/mockery": "^1.6.10",
         "orchestra/testbench": "^8.14 || ^9.0",
-        "phpunit/phpunit": "^10.0",
-        "spatie/laravel-ray": "^1.37"
+        "phpunit/phpunit": "^10.5.35",
+        "spatie/laravel-ray": "^1.37",
+        "spatie/ray": "^1.41.2"
```

Comment:
Both `spatie/ray` and `spatie/laravel-ray` have releases to address deprecations with PHP 8.4.

I've added `spatie/ray` as a dependency so we can pin down the minimum version requirement, otherwise the `lowest-stable` tests will fail.